src/lib/mina_base/user_command.ml

```diff
@@ -362,8 +366,7 @@ module Valid = struct
 end
 
 module For_tests = struct
-  let check_verifiable (t : Verifiable.t) : Valid.t Or_error.t =
-    let signature_kind = Mina_signature_kind.t_DEPRECATED in
+  let check_verifiable ~signature_kind (t : Verifiable.t) : Valid.t Or_error.t =
```

Comment:
I think we can use `signature_kind:Testnet` here and not have a `~signature_kind` parameter.

This functions is defined within `For_tests`, so should never be used in production code (and `Mainnet` signature kind should never occur outside of production code).